### DISCUSSION TITLE
error: there is no package called ‘bit’

### DISCUSSION THREAD

**Discussion by melissaszy** [FrederickHuangLin/ANCOMBC#122] [General]
Hello all, 

I updated my R and R studio recently. I cannot load the ANCOMBC package now. This is the error message I get:

> library("ANCOMBC")
Error: package or namespace load failed for ‘ANCOMBC’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 there is no package called ‘bit’

the versions are:
R: R version 4.2.2 (2022-10-31 ucrt) -- "Innocent and Trusting"
Rstudio: RStudio 2022.07.2+576 "Spotted Wakerobin" Release (e7373ef832b49b2a9b88162cfe7eac5f22c40b34, 2022-09-06) for Windows
ANCOMBC: version 2.0.1

May I ask how I can fix this?

Cheers and thank you for this fantastic package!
Melissa

**Comment by FrederickHuangLin** [comment]
Hi @melissaszy,

What is your current Bioconductor version? 

Best,
Huang

**Comment by melissaszy** [comment]
Dear Huang,

I managed to solve this issue. Previously, I tried to solve it by downloading "bit" package and selecting "yes" when there is a prompt asking me if I want to download dependencies. This did not solve the problem. However, I next tried to download "bit" package and selecting "no" to download dependencies. Somehow, when I don't download dependencies, the package could be downloaded properly and then library("ANCOMBC") does not give an error message anymore.

My ‘BiocManager’ is version 1.30.19

Cheers,
Mel

**Comment by FrederickHuangLin** [comment]
I am glad the problem was solved :)

Cheers,
Huang
